docs/README.mdx

```diff
@@ -6,62 +6,47 @@ import OasisSvg from "@site/static/img/OasisGraphic.svg?react";
 import GetInvolvedSvg from "@site/static/img/GetInvolvedGraphic.svg?react";
 import BuildSvg from "@site/static/img/BuildGraphic.svg?react";
 import DevelopCoreSvg from "@site/static/img/DevelopCoreGraphic.svg?react";
-import NodeseSvg from "@site/static/img/NodesGraphic.svg?react";
+import NodesSvg from "@site/static/img/NodesGraphic.svg?react";
 
 # Getting Started
 
-<div className='homepage-container'>
-  <DocCardSection
-    title="Use Oasis"
-    path="/general/"
-    description="This introductory part contains general overview of the Oasis Network such as
+<div className="homepage-container">
+
+<DocCardSection
+  title="Use Oasis"
+  path="/general/"
+  itemPaths={[
+    "/general/oasis-network/",
+    "/general/manage-tokens/",
+    "/general/manage-tokens/cli/",
+  ]}
+  svg={
+    <OasisSvg className="animatedSvg" />
+  }
+>
+  This introductory part contains general overview of the Oasis Network such as
   the distinction between the consensus layer and different ParaTimes. It
   also covers wallets and other tools for managing your assets across the Oasis
-  chains and how to use unique Oasis features."
-    itemPaths={[
-      "/general/oasis-network/",
-      "/general/manage-tokens/",
-      "/general/manage-tokens/cli/",
-    ]}
-    svg={
-      <OasisSvg className="animatedSvg" />
-    }
-/>
+  chains and how to use unique Oasis features.
+</DocCardSection>
 
 <DocCardSection
   title="Get Involved"
   path="/get-involved/"
-  description="Contains information on official channels to get in touch with the Oasis Network developers and how to contribute to the network."
   itemPaths={[
     "/get-involved/",
     "https://oasisrose.garden",
     "/get-involved/run-node",
     "/get-involved/oasis-core",
   ]}
   svg={<GetInvolvedSvg className="animatedSvg" />}
-/>
+>
+  Contains information on official channels to get in touch with the Oasis Network developers and how to contribute to the network.
+</DocCardSection>
```

Comment:
Converting to children to improve github markdown preview:
| Before | After |
| --- | --- |
|   <img width="893" height="1105" alt="github com_oasisprotocol_docs_tree_main_docs" src="https://github.com/user-attachments/assets/7a0a4d8d-ae5c-4fa6-b952-be9df8d25806" />     |     <img width="893" height="1281" alt="github com_oasisprotocol_docs_tree_lw_homepage-refactor_docs" src="https://github.com/user-attachments/assets/d7fdfd7b-c6ca-470c-9139-6b461d443b2e" />   |